package.json

```diff
@@ -6,6 +6,8 @@
     "@testing-library/jest-dom": "^4.2.4",
     "@testing-library/react": "^9.3.2",
     "@testing-library/user-event": "^7.1.2",
+    "big.js": "^5.2.2",
```

Comment:
In this project Microverse recommends you use `big-js` not `big.js`
![image](https://user-images.githubusercontent.com/31897434/77597489-5fba9280-6eff-11ea-9597-53431780743b.png)
Example:
![image](https://user-images.githubusercontent.com/31897434/77597541-8082e800-6eff-11ea-8fcb-a2d0e33a6610.png)